examples/repo_example/basic_repo.py

```diff
@@ -0,0 +1,397 @@
+"""
+A TUF repository example using the low-level TUF Metadata API.
+
+
+As 'repository_tool' and 'repository_lib' are being deprecated, repository
+metadata must to be created and maintained *manually* using the low-level
+Metadata API. The example code in this file shall serve as temporary
+replacement until a new repository tool is available.
```

Comment:
WDYT to:
```suggestion
Metadata API. The example code in this file demonstrates how to
implement similar functionality to that of the legacy 'repository_tool'
and 'repository_lib' until a new repository implementation is available.
```

Indicates it's an example (and that a user should think before copy/paste verbatim) and doesn't promise to provide a new repository *tool*.